{"trial_id": "01", "hyperparameters": {"space": [{"class_name": "Float", "config": {"name": "dropout_rate", "default": 0.1, "conditions": [], "min_value": 0.1, "max_value": 0.5, "step": 0.1, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "reg_rate", "default": 0.0001, "conditions": [], "min_value": 0.0001, "max_value": 0.001, "step": null, "sampling": "log"}}, {"class_name": "Int", "config": {"name": "first_layer_units", "default": null, "conditions": [], "min_value": 64, "max_value": 128, "step": 16, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 0.0001, "conditions": [], "min_value": 0.0001, "max_value": 0.01, "step": null, "sampling": "log"}}], "values": {"dropout_rate": 0.5, "reg_rate": 0.00013631886043442121, "first_layer_units": 128, "learning_rate": 0.002357749114585103}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"d:\\FIU\\CAI 4203\\Final Project\\projectEnv\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"d:\\FIU\\CAI 4203\\Final Project\\projectEnv\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 239, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"d:\\FIU\\CAI 4203\\Final Project\\projectEnv\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 314, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"d:\\FIU\\CAI 4203\\Final Project\\projectEnv\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner.py\", line 233, in _build_and_fit_model\n    results = self.hypermodel.fit(hp, model, *args, **kwargs)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"d:\\FIU\\CAI 4203\\Final Project\\projectEnv\\Lib\\site-packages\\keras_tuner\\src\\engine\\hypermodel.py\", line 149, in fit\n    return model.fit(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"d:\\FIU\\CAI 4203\\Final Project\\projectEnv\\Lib\\site-packages\\keras\\src\\utils\\traceback_utils.py\", line 122, in error_handler\n    raise e.with_traceback(filtered_tb) from None\n  File \"d:\\FIU\\CAI 4203\\Final Project\\projectEnv\\Lib\\site-packages\\keras\\src\\utils\\traceback_utils.py\", line 124, in error_handler\n    del filtered_tb\n        ^^^^^^^^^^^\nValueError: Exception encountered when calling LSTMCell.call().\n\n\u001b[1mDimensions must be equal, but are 20 and 10 for '{{node sequential_1/lstm_1/lstm_cell_1/MatMul}} = MatMul[T=DT_FLOAT, grad_a=false, grad_b=false, transpose_a=false, transpose_b=false](sequential_1/lstm_1/lstm_cell_1/mul, sequential_1/lstm_1/lstm_cell_1/split)' with input shapes: [?,20], [10,128].\u001b[0m\n\nArguments received by LSTMCell.call():\n  \u2022 inputs=tf.Tensor(shape=(None, 20), dtype=float32)\n  \u2022 states=('tf.Tensor(shape=(None, 128), dtype=float32)', 'tf.Tensor(shape=(None, 128), dtype=float32)')\n  \u2022 training=True\n"}